features/install-wp-tests.feature

```diff
@@ -117,7 +117,7 @@ Feature: Scaffold install-wp-tests.sh tests
       """
     And the /tmp/behat-wordpress/wp-includes/version.php file should contain:
       """
-      -alpha-
+      -
```

Comment:
@miya0001 Did you mean to modify this as well?

Reply:
Yes :)
Test was broken because the version number of the trunk is like following for now while there is the beta release.

```
$wp_version = '4.9-beta1-41801';
```

I guess it will be useful if we have a context which tests regex like following.

```
Then the /path/to/file file should match /regex/
```

I'll try to write it.